src/Debugger.jl

```diff
@@ -95,24 +95,35 @@ function _make_frame(mod, arg)
     end
 end
 
-_check_is_call(arg) = isexpr(arg, :call) || throw(ArgumentError("@enter and @run should be applied to a function call"))
+function _preprocess_enter(__source__, ex)
+    if isexpr(ex, :call)
```

Comment:
This still catches the cases like `@enter 1 .+ 2`.  What's the best way to detect it?  `startswith(string(ex.args[1]), ".")`?